### PR TITLE
Enable CircleCI

### PR DESCRIPTION
**Pull request by jwasinger**
Hopefully transitioning to CircleCI will be better than Travis

### CODE REVIEW

**Comment by jwasinger** [comment]
I'm going to copy over logic from the travis build, then let's merge this.

**Comment by jwasinger** [comment]
Suggesting we merge this and disable travis.

File: circle.yml [hunk 1]
```diff
@@ -35,12 +35,24 @@ defaults:
       name: "Run Ethereum state tests"
       command: |
         echo "ran the state tests."
+  cli-tests: &cli-tests
+    run:
+      name: "basic CLI validation"
+      command: |
+        ./bin/evm2wasm.js asdfdsdf --trace
```
Comment (axic) [inline circle.yml]: Please have it has two tasks as in travis and with proper inputs :)

File: circle.yml [hunk 2]
```diff
@@ -39,7 +44,7 @@ defaults:
     run:
       name: "basic CLI validation"
       command: |
-        ./bin/evm2wasm.js asdfdsdf --trace
+        ./bin/evm2wasm.js 600160020200 --trace
```
Comment (axic) [inline circle.yml]: Do these have `set -e` enabled? e.g. stopping on any error

**Comment by axic** [comment]
Also can you squash this down to a single commit?

**Comment by jwasinger** [comment]
Yes

File: circle.yml (hunk 2 above)
Comment (jwasinger) [reply circle.yml]: `./bin/evm2wasm.js asdfdsdf --trace` this doesn't produce an error.

Comment (jwasinger) [reply circle.yml]: https://github.com/ewasm/evm2wasm/issues/223

**Pull request merged**